sbin/auto_rainbow_grades.py

```diff
@@ -54,6 +58,7 @@
 # Configure variables
 semester = sys.argv[1]
 course = sys.argv[2]
+source = sys.argv[3]
```

Comment:
we could add a little error checking here...  throw an exception if source isn't one of the 2 allowed strings (submitty_gui or submitty_daemon)